MetaMask/metamask-extension
# fix/BannerBase to TS

## Explanation

- Fixes #19181 

## Screenshots/Screencaps



### Before

https://github.com/MetaMask/metamask-extension/assets/79097544/2a2f8168-4875-4632-bff1-4199bcc8729a

### After

https://github.com/MetaMask/metamask-extension/assets/79097544/99c53b48-be00-47af-b24a-7b36b8043394

![image](https://github.com/MetaMask/metamask-extension/assets/79097544/c866b21c-4c3d-409f-a0b4-2e4e2d0abd84)

## Manual Testing Steps

- Run `yarn storybook` on this branch
- Check controls for `BannerBase`
- `yarn jest ui/components/ocmponent-library/banner-base`

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
